Tools/WebKitTestRunner/ios/UIScriptControllerIOS.h

```diff
@@ -187,6 +190,8 @@ class UIScriptControllerIOS final : public UIScriptControllerCocoa {
 #if HAVE(UI_TEXT_SELECTION_DISPLAY_INTERACTION)
     UITextSelectionDisplayInteraction *textSelectionDisplayInteraction() const;
 #endif
+
+    JSObjectRef selectionCaretViewRect(id<UICoordinateSpace>) const;
```

Comment:
Optional nit: place this around the other `selectionCaretViewRect*` functions?

Reply:
Sure!

Reply:
Yeah that makes sense, I was going to suggest `private`-ing it, then I see everything in this class is `private` anyway!